include/crm/common/ordering.h

```diff
@@ -58,6 +58,16 @@ enum pe_ordering {
     //! If 'first' is required, 'then' action for instance on same node is
     pcmk_ord_first_implies_then_instance    = (1U << 10),
 
+    /*!
+     * Disable ordering if first action is unrunnable but for an already-active
+     * resource, otherwise 'then' is runnable only if first action is runnable.
+     * This is used to order a bundle replica's start of its container before
+     * a probe of its remote connection resource, in case the connection uses
+     * the REMOTE_CONTAINER_HACK to replace the connection address with where
+     * the container is running.
```

Comment:
Sort of, but this is only used with probe as then, so it's not relevant

Reply:
In practice, yeah, I was just looking at what the flag actually does/is capable of doing. Both this and `pcmk_ord_unrunnable_first_blocks` make this identical call:
```
changed |= update(then->rsc, first, then, node, first_flags,
                              pcmk_action_runnable, pe_order_runnable_left,
                              data_set);
```

which clears runnable and migratable from `first` if `pcmk_action_runnable` is (un)set in certain flag groups.

Doesn't really matter since it's irrelevant for the *intended* use case of this flag.